payment/tests.py

```diff
@@ -0,0 +1,107 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+
+from payment.models import Payment
+from payment.serializers import PaymentSerializer
+
+
+class PaymentModelTestCase(TestCase):
+    def setUp(self):
+        Payment.objects.create(
+            status="PAID",
+            type="PAYMENT",
+            borrowing_id=1,
+            session_url="https://example.com",
+            session_id="test_session_id",
+            money_to_pay=100.00,
+        )
+
+    def test_payment_str_representation(self):
+        payment = Payment.objects.get(borrowing_id=1)
+        self.assertEqual(str(payment), "Paid - Payment - Borrowing ID: 1")
+
+
+class PaymentSerializerTestCase(TestCase):
+    def setUp(self):
+        self.payment_data = {
+            "status": "PENDING",
+            "type": "FINE",
+            "borrowing_id": 2,
+            "session_url": "https://test.com",
+            "session_id": "test_session_id_2",
+            "money_to_pay": 50.00,
+        }
+        self.serializer = PaymentSerializer(data=self.payment_data)
+
+    def test_serializer_valid(self):
+        self.assertTrue(self.serializer.is_valid())
+
+    def test_serializer_save(self):
+        self.assertTrue(self.serializer.is_valid())
+        self.serializer.save()
+        payment = Payment.objects.get(borrowing_id=2)
+        self.assertEqual(payment.status, "PENDING")
```

Comment:
maybe it's better to assign a wrong `self.payment_data` without `money_to_pay` for example. Then rewrite first test to be `test_serializer_not_valid`. Then add `money_to_pay` in the second test and just save it